models/src/main/java/com/vimeo/networking2/FacebookConfiguration.kt

```diff
@@ -1,20 +1,25 @@
 package com.vimeo.networking2
+import com.squareup.moshi.Json
```

Comment:
formatting issue here, and a similar one exists in other files, might want to run the formatter over all files just to be in the clear.

Reply:
I ran the formatter on all the files in the models module.